app/package.json

```diff
@@ -148,7 +149,7 @@
     "@babel/preset-typescript": "^7.15.0",
     "@electron/notarize": "^2.1.0",
     "electron": "^27.0.4",
-    "electron-builder": "^24.0.0",
+    "electron-builder": "^24.9.1",
```

Comment:
I think package-lock.json needs to be committed too?

Reply:
Good catch! Thanks.